Dockerfiles/manifests/agent-only/agent-services.yaml

```diff
@@ -13,3 +14,24 @@ spec:
     - port: 5005
       name: agentport
       protocol: TCP
+---
+# This file has been generated by `helm template datadog-agent datadog/datadog` from datadog/templates/agent-services.yaml. Please re-run `generate.sh` rather than modifying this file manually.
+apiVersion: v1
+kind: Service
+metadata:
+  name: datadog
+  namespace: default
+  labels:
+    app: "datadog"
+    chart: "datadog-2.28.11"
+    release: "datadog"
+    heritage: "Helm"
+spec:
+  selector:
+    app: datadog
+  ports:
+    - protocol: UDP
+      port: 8125
+      targetPort: 8125
+      name: dogstatsd
+  internalTrafficPolicy: Local
```

Comment:
I think it will be compatible only with k8s 1.22

Reply:
I’ve juste added `--kube-version 1.21` to `helm template` to skip the generation of objects that might not be supported on too old versions of k8s: https://github.com/DataDog/datadog-agent/pull/10393/commits/a772265694ea77c45a8aa01bc669833f291033c4.